unstructured/ingest/connector/weaviate.py

```diff
@@ -0,0 +1,116 @@
+import json
+import os
+import typing as t
+from dataclasses import dataclass
+
+from unstructured.ingest.interfaces import (
+    BaseConnectorConfig,
+    BaseDestinationConnector,
+    BaseIngestDoc,
+    WriteConfig,
+)
+from unstructured.ingest.logger import logger
+from unstructured.utils import requires_dependencies
+
+BATCH_SIZE = 100
```

Comment:
How was this batch size chosen? Should this be something configurable from the user with a default set?